### PR TITLE
Edited on chrome

### PR DESCRIPTION
**Pull request by erezfree29**
Page looks ok on google chrome.

when published to Github pages the page displaying diffrently and also the logo is not displaying.

Earlier on I used firefox to edit the file and when I published the github pages the page displayed other elements diffrently and my positioning looked out of place.

now if I open the file using firefox the page displays diffrently than in google chrome and the logo is not displayed as well
This is how the page looks on google chrome 

screenshotwensday

### CODE REVIEW

**Comment by erezfree29** [comment]
For some reason the page looks diffrent on github pages and on firefox than it looks chrome 
this is the screenshot from chrome 
screenshotwensday
on github pages the logo is also not displyaing

File: index.html [hunk 1]
```diff
@@ -1,119 +1,634 @@
...
+          </svg>
+          
+          <svg  width="1em" height="1em" viewBox="0 0 16 16" class="bi bi-search search" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+            <path fill-rule="evenodd" d="M10.442 10.442a1 1 0 0 1 1.415 0l3.85 3.85a1 1 0 0 1-1.414 1.415l-3.85-3.85a1 1 0 0 1 0-1.415z"/>
+            <path fill-rule="evenodd" d="M6.5 12a5.5 5.5 0 1 0 0-11 5.5 5.5 0 0 0 0 11zM13 6.5a6.5 6.5 0 1 1-13 0 6.5 6.5 0 0 1 13 0z"/>
+          </svg>
+           
+            <div class="space"> SPACE & COSMOS</div>
+        </div>
+        <a href="#"><img  alt="logo" class="logo" src="/images/logo.png"></a>
```
Comment (MuriloRoque) [inline index.html]: - [ ] Your relative path for the image is wrong, it should be `./images/logo.png` 😉

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Changes requested 🔴 

### Hello @erezfree29 👋 

### I can see that you put a lot of effort into this project, it looks really nice 👏 

### Please try my suggestion below in your code and see if your page will load on GH Pages.

Kindly make the requested changes and submit for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello again @erezfree29 :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**